JS_2_names.js

```diff
@@ -0,0 +1,21 @@
+var arrUserNames = [];
+
+for (var i = 0; i < 5 ; i++) {
+  arrUserNames[i] = prompt('Введите имя пользователя');
```

Comment:
It makes sense to change user prompt for different iterations, e.g. `Enter user name #N`, so the user could understand your program is doing something.

Reply:
done